_posts/2019-09-08-elixirconf-2019-review.md

```diff
@@ -0,0 +1,58 @@
+
+* Day 1: Harnessing the Real-Time Web with Phoenix, Channels, and Presence
+  * Slide deck
+  * Complete code
+* Day 2: Building Bulletproof Real-Time Applications with Phoenix, LiveView, and StreamData
+  * Slide deck on LiveView
+  * Complete code for LiveView features
+  * Slide deck on Property-based testing with StreamData
+  * Complete code for testing features
+
+
+## Talks We Enjoyed
+
+There were so many exciting talks to check out, but with a multi-track schedule, we sadly couldn't catch them all. Here are a few that stood out to us in particular.
+
+### Jose Valim's Thursday Evening Keynote
+
+Just because Elixir is now feature-complete doesn't mean that the core team doesn't have some exciting and ambitious goals for the next year! Jose discussed the present and future of `Mix.Config`, including some thoughts on build time vs. run time configuration. He also made an exciting announcement--Phoenix 1.5 will ship with a native Telemetry module that makes it easy to configure your app's Telemetry metrics and report them to the third party service of your choosing (New Relic, Statsd, etc.). The Elixir community can expect a new version released every six months. One feature that we're excited for is ExUnit pattern diffing––making it easy to see exactly *what* doesn't match when ExUnit tells you that your match failed. Learn more about the future of Elixir by checking out the full talk [here](https://www.youtube.com/watch?v=oUZC1s1N42Q).
+
+### Chris McCord's Friday Evening Keynote: LiveView in the Wild
+
+This past year, Chris McCord and other core Phoenix maintainers gave us LiveView––a technology for writing rich, real-time UX with server-rendered HTML. Here, Chris walks us through the powerful features of LiveView and gives us a look at how it behaves "in the wild", especially compared to traditional SPAs (Single Page Applications). LiveView's first release contains a set of exciting and powerful features––including JavaScript hooks, Live navigation and LiveView test. Learn more about this incredibly powerful set of tooling by checking out the full talk [here](https://www.youtube.com/watch?v=XhNv1ikZNLs).  
+
+### Contracts for Building Reliable Systems - Chris Keathley
+
+Chris Keathley's talk asked the question: "How can we build systems that withstand change?", and answered it with his new library, [Norm](https://github.com/keathley/norm). Norm was developed in response to the challenges that many of us face coordinating across teams and services and ensuring that we don't break APIs will encouraging and allowing for growth. By embracing contract-driven design with the help of the [ExContract](https://hexdocs.pm/ex_contract/readme.html) testing library, and by leveraging Norm to enforce data specification, we can embrace change and mitigate breakage. Learn more about how Elixir tooling helps us promote contract-driven design for distributed architecture by checking out the full talk here.
```

Comment:
I think in place of "APIs will encouraging and allowing for growth." you wanted "while encouraging"

Reply:
The link is missing for the talk